eucaconsole

```diff
@@ -35,18 +36,10 @@ def daemonize(func=None):
     if procid == -1:
         sys.exit(1)
 
-    # Close open file descriptors
-    for fd in (0, 1, 2):
-        try:
-            os.close(fd)
-        except OSError:
-            pass
-
-    print("about to point fds to null")
-    # Point descriptors to /dev/null
-    os.open("/dev/null", os.O_RDWR)
-    os.dup(0)
-    os.dup(0)
+    # output descriptors point to log file
+    f = os.open(log_file, os.O_WRONLY|os.O_APPEND)
```

Comment:
This is going to break if the file doesn't already exist, but I guess we're stuck in that situation for the main log file anyway.  Hopefully systemd-journald will help make our logging less fragile when we move to RHEL 7.